hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/AbstractFacade.java

```diff
@@ -134,7 +134,7 @@ public String buildSortString(Set<? extends SortBy> sortBy) {
     if (!sort.hasNext()) {
       return "";
     }
-    StringBuilder c = new StringBuilder("ORDER BY " + OrderBy(sort.next()));
+    StringBuilder c = new StringBuilder(" ORDER BY " + OrderBy(sort.next()));
```

Comment:
@kai-chi I guess you hit some corner case and had to add the whitespace?

Reply:
the corner case was that I didn't add a space at the end of my query. It's better to make AbstractFacade with the spaces. in the worst case scenario we'll have two spaces in a query next to each other